network-uri-json.cabal

```diff
@@ -15,7 +15,7 @@ copyright:           (c) 2017 Alex Brandt
 category:            Network
 build-type:          Simple
 cabal-version:       >=1.10
-tested-with:         GHC >= 7.6 && < 8.2.1 || > 8.2.1 && < 9.0
+tested-with:         GHC >= 7.6 && < 8.8.3
```

Comment:
I've updated this, but I didn't understand the syntax it had before, why was the second condition after `||`? It wasn't correct.

Reply:
I was excluding version 8.2.1 but I don't recall why.  I'd say leave the exclusion in and don't specify the minor version in the gop GHC version.  Looking at it, the previous version looks like what I'd go with and we probably just need to run:

```
haskell-ci --hlint regenerate network-uri-json.cabal
```

If you don't mind adding that, I'll see about the travis failures (I expect some version mismatches with network-arbitrary but we'll see).

Reply:
well, then `GHC >= 7.6 && < 8.2.1 || > 8.2.1 && < 9.0` was alright, I'll revert back to it